packages/sentry/README.md

```diff
@@ -0,0 +1,148 @@
+
+package provides one super basic adapters :
+
+- The `SentryMonitoringAdapter` logs errors and measure data in the Sentry platform.
+
+# Basic Usage
+## Set Up
+In order to configure the sentry inside your application, you first need to instantiate the Sentry Adapter and then wrap your root component in the `MonitoringProvider` and passing it the adapter as a prop :
+
+```typescript jsx
+import { SentryMonitoringAdapter } from '@monkvision/sentry';
+import { MonitoringProvider } from '@monkvision/monitoring';
+
+const adapter = new SentryMonitoringAdapter({
+  dsn: '',
+  environment: '',
+  debug: true / false,
+  tracesSampleRate: 0.025,
+  release: '',
+});
+
+const container = document.getElementById('root');
+render((
+  <MonitoringProvider adapter={adapter}>
+    <App/>
+  </MonitoringProvider>
+), container);
+```
+
+## useMonitoring hook
+Once you have wrapped up your application in the `MonitoringProvider` component, you can now access all of the
+monitoring features in your app components using the `useMonitoring` hook :
+
+```typescript jsx
+import { useMonitoring } from '@monkvision/monitoring';
+
+function MyCustomComponent() {
+  const {
+    setUserId,
+    log,
+    handleError,
+    createTransaction,
+  } = useMonitoring();
+}
+```
+
+# API
+## Monitoring Methods
+This section describes the methods available in the `SentryMonitoringAdapter` interface.
+
+#### setUserId
+```typescript
+setUserId: (id: string) => void
+```
+This method defines the current user using the application. Users are identified by a unique string ID.
+
+#### log
+```typescript
+log: (msg: string, context?: LogContext | Severity) => void
+```
+This method logs messages in Sentry. An optional context can be provided that can contain :
+- A log severity level (default: `info`)
+- Extra data to send with the logs
+- Tags associated with the log
+
+#### handleError
+```typescript
+handleError: (err: Error | string, context?: Omit<LogContext, 'level'>) => void
+```
+This method reports errors in Sentry which can be displayed in Sentry dashboard. An optional context can be provided that can contain :
+- Extra data to send with the error
+- Tags associated with the error
+
+#### createTransaction
+```typescript
+createTransaction: (context: TransactionContext) => Transaction
+```
+This method creates a new transaction that can be used to measure performances, report metrics etc. A Transaction object
+has the following structure :
+
+```typescript
+export interface Transaction {
+  // Set a tag in the transaction
+  setTag: (name: string, value: string) => void;
+  // Start a performance measurement
+  startMeasurement: (name: string, data?: { [key: string]: number | string }) => void;
+  // Stop a given measurement
+  stopMeasurement: (name: string) => void;
+  // Complete the transaction
+  finish: (status?: string) => void;
+}
+```
+
+## Adapters
```

Comment:
You can remove this header : You can have the `SentryMonitoringAdapter` section one layer above and you can remove one hashtag from the subtitles of the `SentryMonitoringAdapter` section